### PR TITLE
SFR-679 Removing Emulsify, by removing Patternlab

### PR DESCRIPTION
**Pull request by hokei**
[ResearchNow Jira Ticket](http://jira.nypl.org/browse/SFR-679)

## **This PR does the following:**
- Removes Patternlab and configures Storybook to run both Twig and React components. 
- Updates READMEs to reflect this
- Modifies twig tests to remove the necessity of PHP

### CODE REVIEW

**Comment by hokei** [comment]
So this CR is mostly configuration and pretty dense.  Might be easier to do in person!  
There are files in twig/_annotations, twig/_data, etc that we might also be able to take out, too!  Will defer to you for that!

File: storybook/.storybook-react/webpack.config.js
Comment (helenvholmes) [inline storybook/.storybook-react/webpack.config.js]: This is no longer needed for the storybook-react since we have storybook-html, correct?

**Review by helenvholmes — APPROVED** [review]
So damn pleased. 🥳

File: storybook/.storybook-react/webpack.config.js
Comment (hokei) [reply storybook/.storybook-react/webpack.config.js]: Yeah, I could take this out.  I'll do that before merging

**Pull request merged**
